charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -5,986 +5,6 @@
 #
```

Comment:
it looks like that `charts/openyurt/templates/yurt-manager-auto-generated.yaml` is not needed because clusterrole and webhooks file are dependent files.

but after execute `make manifests` command, `charts/openyurt/templates/yurt-manager-auto-generated.yaml` will be generated.